product_docs/docs/pgd/5/manual/02-install-postgres.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Step 2 - Installing Postgres
+navTitle: Installing Postgres
+deepToC: true
+---
+
+## Installing Postgres
+
+You will need to install Postgres on all the hosts.
+
+You will need to have an EDB account and use the [EDB Repos 2.0](https://www.enterprisedb.com/repos) page.
+Select your platform and Postgres edition. 
+You will be presented with 2 steps of instructions, the first covering how to install the required package repository and the second covering how to load the packages from that repository.
+
+Run both steps.
+
+## Worked example
+
+In our example, we will be installing EDB Postgres Advanced Server 16 on Rocky Linux 9 (a RHEL compatible Linux).
```

Comment:
For consistency, let's stick to pretending we're using RHEL.

Reply:
Fixed.